### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by bebehei**
Add an issue template and give automatic hints to the users to provide better details.

Any comments?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!-- *Delete this comment before submitting*
```
Comment (tsipinakis) [inline .github/ISSUE_TEMPLATE.md]: Why does this need to be deleted, I thought HTML comments don't show up in issues anyway?

Comment (bebehei) [reply .github/ISSUE_TEMPLATE.md]: Yes, it actually makes no sense. My intention was, that the users don't write their message content into the comment by accident.

Comment (tsipinakis) [reply .github/ISSUE_TEMPLATE.md]: Closing the second comment at the end should help to avoid that :)

**Comment by tsipinakis** [comment]
Not sure why I've delayed this - LGTM

**Pull request merged**